reassembler_test.go

```diff
@@ -105,6 +105,13 @@ type results struct {
 	events  []eventMeta
 }
 
+func testPayloadForMessage(msg *auparse.AuditMessage) map[string]interface{} {
+	return map[string]interface{}{
+		"seq": msg.Sequence,
+		"typ": msg.RecordType,
+	}
+}
```

Comment:
Caller before callee.

Reply:
Can you place the helper below the calling function in source order. We don't have C compiler constraints and context first makes code easier to comprehend.

Reply:
Moved after the calling function, gave it a better name.